website/www/site/layouts/shortcodes/documentation/capability-matrix-single.html

```diff
@@ -0,0 +1,101 @@
+{{ end }}
+
+
+<div class="information-container">
+    <div>
+        <h5>How to read the tables</h5>
+        <table class="read-tables">
+            <tr>
+                <th></th>
+                <th class="border-left">Tools we are comparing</th>
+            </tr>
+            <tr class="border-top">
+                <td class="border-right">Properties</td>
+                <td style="border-left: 3px solid #ff6d05">
+                    Does this tool have this property?<span>Yes/Partially/No</span>
+                </td>
+            </tr>
+        </table>
+    </div>
+    <div class="second-container">
+        <h5>What do those signs mean?</h5>
+        <div class="row">
+            <div class="box white">&#x2713;</div>
+            <div>Yes</div>
+        </div>
+        <div class="row">
+            <div class="box partial">~</div>
+            <div>Partially</div>
+        </div>
+        <div class="row">
+            <div class="box gray">&#x2715;</div>
+            <div>No</div>
+        </div>
+    </div>
+</div>
+{{ range $category := $cap_data.categories }}
+    <h4>{{ $category.description }}</h4>
+    <div class="table-container">
+        <div class="table-left">
+            <table>
+                <tr>
+                    <th></th>
+                </tr>
+                {{ range $row := $category.rows }}
+                    <tr>
+                        <th>{{ $row.name }}</th>
+                    </tr>
+                {{ end }}
+            </table>
+        </div>
+        <div class="table-right">
+            <div id="{{ $category.description | urlize }}" class="table-headers">
+                <table>
+                    <tr>
+                        {{ range $column := $cap_data.columns }}
+                            <th>{{ $column.name }}</th>
+                        {{ end }}
+                    </tr>
+                </table>
+            </div>
+            <div id="{{ $category.description | urlize }}" class="table-center">
+                <table>
+                    {{ range $row := $category.rows }}
+                        <tr>
+                            {{ range $val := $row.values }}
+                                <td style='background-color:#{{ if eq $val.l1 "Yes" }}{{ index $category "color-y" }}{{ else if eq $val.l1 "Partially" }}{{ index $category "color-p" }}{{ else }}{{ index $category "color-n" }}{{ end }};border-color:#{{ if eq $val.l1 "Yes" }}{{ index $category "color-yb" }}{{ else if eq $val.l1 "Partially" }}{{ index $category "color-pb" }}{{ else }}{{ index $category "color-nb" }}{{ end }}'>
+                                    {{ partial "documentation/capability-matrix-row" (dict "type" $cap_view "val" $val) }}
+                                </td>
+                            {{ end }}
+                        </tr>
+                    {{ end }}
+                </table>
+            </div>
+        </div>
+    </div>
```

Comment:
The way the capability matrix scrolls seems buggy. It's like the header and the content don't scroll at the same rate, what makes it hard to line them up. After I scrolled a little bit the table looks like this:
![image](https://user-images.githubusercontent.com/675055/106936846-eddfd780-66d1-11eb-9675-96747df24298.png)

Where the headers are almost on the line between the columns. Do you know what's going on here?

Reply:
hmm it's weird these headers and columns are synchronized with js, there is a little delay but something like above never happend before, not in my browser localy and not during the tests also, I will try to repeat this situation. Do you think it can be because of the way the build is made here?

Reply:
Okay it's happening when we open a console in browser, I still don't know why but until then everything goes smoothly

Reply:
Okay I changed the function, there is little difference, pls check if u can repeat situation from above

Reply:
It's still happening for me. Note I'm using Chrome on Linux if that helps.

Also if I click and drag the scrollbar, only the content moves, the headers stay where they started, until I do a horizontal scroll again.

Reply:
Not sure what was going on before, but this seems to be resolved for me now. I restarted my browser, and cleared the cache. Also made sure not to have developer tools open when scrolling. Sorry for the false alarm.